app/Services/UserService.php

```diff
@@ -46,4 +46,27 @@ public function loginUser($data)
         return ['user' => $user, 'token' => $token];
     }
 
+    public function changePassword($userId, $oldPassword, $newPassword)
```

Comment:
here you should pass only $data that will include (email, oldPassword and newPassword)